datasets/udhr/udhr.py

```diff
@@ -46,6 +46,8 @@
 class UDHN(datasets.GeneratorBasedBuilder):
     """Universal Declaration of Human Rights"""
 
+    VERSION = datasets.Version("1.2022.0519")
```

Comment:
Also note that we normally use semantic versioning.

Reply:
Do you have a link to an acceptable scheme?

Reply:
This is the first version with a version number, so bumped to 1.0.0